src/api/migrations/20190621210101-create-sequelize-meta.js

```diff
@@ -0,0 +1,8 @@
+export default {
```

Comment:
@MCFrank16 I thought this table already exists?

Reply:
I am deleting it as @nkpremices removed it also in the migration branch